pkg/lib/gzip/gzip.go

```diff
@@ -127,56 +122,44 @@ func DecompressWithMaxBytes(tarGzPath, destDir string, maxDecompressSize int64)
 			break // End of archive
 		}
 		if err != nil {
-			return fmt.Errorf("failed to read tar header: %w", err)
+			return err
```

Comment:
Any reason for making these error messages less verbose?

Reply:
brought them back in https://github.com/bacalhau-project/bacalhau/pull/4069/commits/d1106e5ca2e8260c42e977785f9f74213769f2ec